charts/mlflow/templates/deployment.yaml

```diff
@@ -66,22 +66,26 @@ spec:
             - name: http
               containerPort: 5000
               protocol: TCP
+          {{- if .Values.livenessProbe.enabled }}
```

Comment:
I'm afraid I have to disagree with making the liveness probe optional. It's best practice to have it for production deployments.